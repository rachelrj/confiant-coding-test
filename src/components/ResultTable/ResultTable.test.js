import ResultTable from "./ResultTable";
import React from 'react';
import renderer from 'react-test-renderer';

test('Renders table', () => {
    const component = renderer.create(
      <ResultTable results={[{
        name: 'name',
        path: 'path',
        html_url: 'url',
      }]} />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
