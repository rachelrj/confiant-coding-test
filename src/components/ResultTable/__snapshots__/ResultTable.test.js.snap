// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders table 1`] = `
<div
  className="MuiPaper-root MuiTableContainer-root MuiPaper-elevation1 MuiPaper-rounded"
>
  <table
    aria-label="simple table"
    className="MuiTable-root"
    role={null}
  >
    <thead
      className="MuiTableHead-root"
      role={null}
    >
      <tr
        className="MuiTableRow-root MuiTableRow-head"
        role={null}
      >
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head"
          scope="col"
        >
          Name
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight"
          scope="col"
        >
          Path
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight"
          scope="col"
        >
          Link
        </th>
      </tr>
    </thead>
    <tbody
      className="MuiTableBody-root"
      role={null}
    >
      <tr
        className="MuiTableRow-root"
        role={null}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
        >
          name
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
        >
          path
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
        >
          <a
            href="url"
            rel="noreferrer"
            target="_blank"
          >
            Link
          </a>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
